internal/boxcli/add.go

```diff
@@ -24,9 +23,9 @@ func AddCmd() *cobra.Command {
 	flags := addCmdFlags{}
 
 	command := &cobra.Command{
-		Use:               "add <pkg>...",
-		Short:             "Add a new package to your devbox",
-		PersistentPreRunE: nix.EnsureInstalled,
+		Use:     "add <pkg>...",
+		Short:   "Add a new package to your devbox",
+		PreRunE: nix.EnsureInstalled,
```

Comment:
Just curious, why this change? can we not have 2 `PersistentPreRunE` ? (one in root and another in the subcommands?)

Reply:
A found it https://github.com/spf13/cobra/issues/252

looks like others were confused by this as well. Good find!

Reply:
This was causing the subcommands ignore the persistentPreRun at root level which contains `-q` flag. This was the bug I couldn't solve yesterday but eventually did 😄